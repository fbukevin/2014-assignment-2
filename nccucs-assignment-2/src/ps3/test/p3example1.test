# General test.  Tests implementation of accessing nodes and children

CreateGraph X
CreateNode n1 7
CreateNode n2 2
ListNodes X
AddNode X n1
AddNode X n2
ListNodes X
AddEdge X n1 n2

ListNodes X
ListChildren X n1
ListChildren X n2