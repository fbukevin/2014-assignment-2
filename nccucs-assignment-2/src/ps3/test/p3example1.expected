# General test.  Tests implementation of accessing nodes and children

created graph X
created node n1 with cost 7
created node n2 with cost 2
X contains:
added node n1 to X
added node n2 to X
X contains: n1 n2
added edge from n1 to n2 in X

X contains: n1 n2
the children of n1 in X are: n2
the children of n2 in X are: